deploy/olm-catalog/0.7.0/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
```

Comment:
Let's remove this file and all other generated files. The repo is not a place to store (CI?) build artifacts.

Reply:
Done